src/core/ai_cache.py

```
@@ -0,0 +1,26 @@
+Core caching functionality for AI operations.
+Moved from app/core/ai_cache.py
+"""
+
+from typing import Any, Optional
+import redis
+from datetime import timedelta
+
+class AICache:
+    def __init__(self, redis_url: str):
+        self.redis = redis.from_url(redis_url)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)